src/components/Text/Text.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem'; // eslint-disable-line import/no-unresolved
```

Comment:
we can add a eslint rule in eslintrc to ignore bem import so we don't have to add this comment each time: `import/no-unresolved: [2, { ignore: ['bem'] }]`
probably there is also a more proper solution with https://github.com/benmosher/eslint-plugin-import/blob/master/README.md#resolver-plugins. but maybe the above would do